2018-06-26 11:43:08.637 10812 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-26 11:43:08.640 10812 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-26 11:44:08.638 10812 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-26 11:44:08.641 10812 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-26 11:45:08.638 10812 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-26 11:45:08.641 10812 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-26 11:45:31.628 10812 DEBUG oslo_concurrency.lockutils [req-1f2ac3f0-e37a-4c0e-9a8f-2e65b1ff4a2a 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Acquired semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-26 11:45:32.089 10812 DEBUG neutron.agent.dhcp.agent [req-1f5b0192-b22b-4394-9606-58d2bf0222c5 - - - - -] Calling driver for network: c488251b-964b-472c-89c5-f93c510ef717 action: enable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-26 11:45:32.090 10812 DEBUG neutron.agent.linux.utils [req-1f5b0192-b22b-4394-9606-58d2bf0222c5 - - - - -] Unable to access /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-26 11:45:32.091 10812 DEBUG neutron.agent.linux.dhcp [req-1f5b0192-b22b-4394-9606-58d2bf0222c5 - - - - -] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-c488251b-964b-472c-89c5-f93c510ef717 on network c488251b-964b-472c-89c5-f93c510ef717 does not yet exist. Checking for a reserved port. _setup_reserved_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1267
2018-06-26 11:45:32.091 10812 DEBUG neutron.agent.linux.dhcp [req-1f5b0192-b22b-4394-9606-58d2bf0222c5 - - - - -] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-c488251b-964b-472c-89c5-f93c510ef717 on network c488251b-964b-472c-89c5-f93c510ef717 does not yet exist. Creating new one. _setup_new_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1288
2018-06-26 11:45:35.416 10812 DEBUG neutron.agent.linux.utils [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:45:35.444 10812 DEBUG neutron.agent.linux.utils [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'add', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:45:35.473 10812 DEBUG neutron.agent.linux.utils [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'sysctl', '-w', 'net.ipv4.conf.all.promote_secondaries=1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:45:35.504 10812 DEBUG neutron.agent.linux.utils [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', 'link', 'set', 'lo', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:45:35.532 10812 DEBUG neutron.agent.linux.utils [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'sysctl', '-w', 'net.ipv6.conf.default.accept_ra=0'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:45:35.550 10812 DEBUG neutron.agent.linux.utils [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', 'link', 'set', 'tapf857e23e-8a', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:45:35.607 10812 DEBUG neutron.agent.linux.utils [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', '-o', 'link', 'show', 'tapf857e23e-8a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:45:35.652 10812 DEBUG neutron.agent.linux.utils [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-06-26 11:45:36.039 10812 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=None, port=tapf857e23e-8a) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-26 11:45:36.040 10812 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Transaction caused no change do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:114
2018-06-26 11:45:36.041 10812 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): AddPortCommand(bridge=br-int, may_exist=False, port=tapf857e23e-8a) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-26 11:45:36.042 10812 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=1): DbSetCommand(table=Interface, col_values=(('type', 'internal'), ('external_ids', {'iface-id': u'f857e23e-8a19-4534-bd37-e7a0f68da19c', 'iface-status': 'active', 'attached-mac': u'fa:16:3e:fc:39:36'})), record=tapf857e23e-8a) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-26 11:45:36.107 10812 DEBUG neutron.agent.linux.utils [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tapf857e23e-8a', 'address', 'fa:16:3e:fc:39:36'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:45:36.118 10812 DEBUG neutron.agent.linux.utils [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:45:36.144 10812 DEBUG neutron.agent.linux.utils [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tapf857e23e-8a', 'netns', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:45:36.191 10812 DEBUG neutron.agent.linux.utils [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', 'link', 'set', 'tapf857e23e-8a', 'mtu', '1450'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:45:36.209 10812 DEBUG neutron.agent.linux.utils [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', 'link', 'set', 'tapf857e23e-8a', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:45:36.258 10812 DEBUG oslo_concurrency.lockutils [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Acquired semaphore "iptables-qdhcp-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-26 11:45:36.260 10812 DEBUG neutron.agent.linux.utils [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'iptables-save'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:45:36.295 10812 DEBUG neutron.agent.linux.utils [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'iptables-restore', '-n'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:45:36.314 10812 DEBUG neutron.agent.linux.iptables_manager [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] IPTablesManager.apply completed with success. 56 iptables commands were issued _apply_synchronized /usr/lib/python2.7/site-packages/neutron/agent/linux/iptables_manager.py:601
2018-06-26 11:45:36.315 10812 DEBUG oslo_concurrency.lockutils [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Releasing semaphore "iptables-qdhcp-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-26 11:45:36.316 10812 DEBUG neutron.agent.linux.utils [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', '-o', 'link', 'show', 'tapf857e23e-8a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:45:36.343 10812 DEBUG neutron.agent.linux.utils [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', 'addr', 'show', 'tapf857e23e-8a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:45:36.361 10812 DEBUG neutron.agent.linux.utils [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', '-4', 'addr', 'add', '11.219.1.2/24', 'scope', 'global', 'dev', 'tapf857e23e-8a', 'brd', '11.219.1.255'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:45:36.425 10812 DEBUG neutron.agent.linux.utils [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', '-4', 'route', 'list', 'dev', 'tapf857e23e-8a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:45:36.442 10812 DEBUG neutron.agent.linux.dhcp [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Setting IPv4 gateway for dhcp netns on net c488251b-964b-472c-89c5-f93c510ef717 to 11.219.1.1 _set_default_route_ip_version /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1164
2018-06-26 11:45:36.444 10812 DEBUG neutron.agent.linux.utils [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', '-4', 'route', 'replace', 'default', 'via', '11.219.1.1', 'dev', 'tapf857e23e-8a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:45:36.468 10812 DEBUG neutron.agent.linux.utils [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', '-6', 'route', 'list', 'dev', 'tapf857e23e-8a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:45:36.485 10812 DEBUG neutron.agent.linux.dhcp [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Cleaning stale devices for network c488251b-964b-472c-89c5-f93c510ef717 _cleanup_stale_devices /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1369
2018-06-26 11:45:36.488 10812 DEBUG neutron.agent.linux.utils [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:45:36.506 10812 DEBUG neutron.agent.linux.utils [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'find', '/sys/class/net', '-maxdepth', '1', '-type', 'l', '-printf', '%f '] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:45:36.534 10812 DEBUG neutron.agent.linux.dhcp [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Building initial lease file: /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/leases _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:623
2018-06-26 11:45:36.536 10812 DEBUG neutron.agent.linux.dhcp [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Done building initial lease file /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/leases with contents:
1530092736 fa:16:3e:fc:39:36 11.219.1.2 * *
 _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:651
2018-06-26 11:45:36.537 10812 DEBUG neutron.agent.linux.dhcp [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Building host file: /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-26 11:45:36.539 10812 DEBUG neutron.agent.linux.dhcp [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Done building host file /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-26 11:45:36.544 10812 DEBUG neutron.agent.linux.utils [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Unable to access /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-26 11:45:36.546 10812 DEBUG neutron.agent.linux.utils [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'dnsmasq', '--no-hosts', '--no-resolv', '--strict-order', '--except-interface=lo', '--pid-file=/var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/pid', '--dhcp-hostsfile=/var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/host', '--addn-hosts=/var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/addn_hosts', '--dhcp-optsfile=/var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/opts', '--dhcp-leasefile=/var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/leases', '--dhcp-match=set:ipxe,175', '--bind-interfaces', '--interface=tapf857e23e-8a', '--dhcp-range=set:tag0,11.219.1.0,static,255.255.255.0,86400s', '--dhcp-option-force=option:mtu,1450', '--dhcp-lease-max=256', '--conf-file=', '--domain=openstacklocal'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:45:36.621 10812 DEBUG neutron.agent.linux.utils [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Unable to access /var/lib/neutron/external/pids/c488251b-964b-472c-89c5-f93c510ef717.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-26 11:45:36.623 10812 DEBUG neutron.agent.linux.utils [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Unable to access /var/lib/neutron/external/pids/c488251b-964b-472c-89c5-f93c510ef717.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-26 11:45:36.625 10812 DEBUG neutron.agent.linux.external_process [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] No process started for c488251b-964b-472c-89c5-f93c510ef717 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-26 11:45:36.625 10812 DEBUG oslo_concurrency.lockutils [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Releasing semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-26 11:45:36.629 10812 DEBUG oslo_concurrency.lockutils [req-1f2ac3f0-e37a-4c0e-9a8f-2e65b1ff4a2a 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Acquired semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-26 11:45:38.699 10812 DEBUG neutron.agent.dhcp.agent [req-6638b1b6-6876-48a5-96a4-045428744042 - - - - -] Calling driver for network: c488251b-964b-472c-89c5-f93c510ef717 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-26 11:45:38.701 10812 DEBUG neutron.agent.linux.dhcp [req-6638b1b6-6876-48a5-96a4-045428744042 - - - - -] Building host file: /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-26 11:45:38.707 10812 DEBUG neutron.agent.linux.dhcp [req-6638b1b6-6876-48a5-96a4-045428744042 - - - - -] Done building host file /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-26 11:45:38.715 10812 DEBUG neutron.agent.linux.utils [req-6638b1b6-6876-48a5-96a4-045428744042 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '14888'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:45:38.750 10812 DEBUG neutron.agent.linux.dhcp [req-6638b1b6-6876-48a5-96a4-045428744042 - - - - -] Reloading allocations for network: c488251b-964b-472c-89c5-f93c510ef717 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-26 11:45:38.752 10812 DEBUG neutron.agent.linux.utils [req-6638b1b6-6876-48a5-96a4-045428744042 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', '-4', 'route', 'list', 'dev', 'tapf857e23e-8a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:45:38.767 10812 DEBUG neutron.agent.linux.utils [req-6638b1b6-6876-48a5-96a4-045428744042 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', '-6', 'route', 'list', 'dev', 'tapf857e23e-8a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:45:38.796 10812 DEBUG neutron.agent.linux.utils [req-6638b1b6-6876-48a5-96a4-045428744042 - - - - -] Unable to access /var/lib/neutron/external/pids/c488251b-964b-472c-89c5-f93c510ef717.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-26 11:45:38.798 10812 DEBUG neutron.agent.linux.utils [req-6638b1b6-6876-48a5-96a4-045428744042 - - - - -] Unable to access /var/lib/neutron/external/pids/c488251b-964b-472c-89c5-f93c510ef717.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-26 11:45:38.799 10812 DEBUG neutron.agent.linux.external_process [req-6638b1b6-6876-48a5-96a4-045428744042 - - - - -] No process started for c488251b-964b-472c-89c5-f93c510ef717 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-26 11:45:38.803 10812 DEBUG oslo_concurrency.lockutils [req-6638b1b6-6876-48a5-96a4-045428744042 - - - - -] Releasing semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-26 11:45:38.805 10812 DEBUG oslo_concurrency.lockutils [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Acquired semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-26 11:45:38.805 10812 DEBUG neutron.agent.dhcp.agent [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Discarding stale port update: admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-26T09:45:32Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-c488251b-964b-472c-89c5-f93c510ef717, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'dc38aae2-7422-48a5-8795-4b6078da6508', u'ip_address': u'11.219.1.2'}], id=f857e23e-8a19-4534-bd37-e7a0f68da19c, mac_address=fa:16:3e:fc:39:36, name=, network_id=c488251b-964b-472c-89c5-f93c510ef717, port_security_enabled=False, project_id=fa8500bb2b5f42e698af7eb9461c6a60, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=fa8500bb2b5f42e698af7eb9461c6a60, updated_at=2018-06-26T09:45:33Z port_update_end /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:409
2018-06-26 11:45:38.806 10812 DEBUG oslo_concurrency.lockutils [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Releasing semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-26 11:45:38.807 10812 DEBUG oslo_concurrency.lockutils [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Acquired semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-26 11:45:38.807 10812 DEBUG neutron.agent.dhcp.agent [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Discarding stale port update: admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-06-26T09:45:32Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-c488251b-964b-472c-89c5-f93c510ef717, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'dc38aae2-7422-48a5-8795-4b6078da6508', u'ip_address': u'11.219.1.2'}], id=f857e23e-8a19-4534-bd37-e7a0f68da19c, mac_address=fa:16:3e:fc:39:36, name=, network_id=c488251b-964b-472c-89c5-f93c510ef717, port_security_enabled=False, project_id=fa8500bb2b5f42e698af7eb9461c6a60, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=fa8500bb2b5f42e698af7eb9461c6a60, updated_at=2018-06-26T09:45:34Z port_update_end /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:409
2018-06-26 11:45:38.808 10812 DEBUG oslo_concurrency.lockutils [req-4e11da2f-59ad-4a0c-973f-7986850c9da3 - - - - -] Releasing semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-26 11:46:08.645 10812 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-26 11:46:08.649 10812 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-26 11:46:15.525 10812 DEBUG oslo_concurrency.lockutils [req-71360d0c-3d9d-4a0e-b864-f8a8a4bc1e81 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Acquired semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-26 11:46:15.526 10812 INFO neutron.agent.dhcp.agent [req-71360d0c-3d9d-4a0e-b864-f8a8a4bc1e81 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-26T09:46:13Z, description=, device_id=cba92f07-3ddc-4e6b-98aa-37bb1d876edc, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'dc38aae2-7422-48a5-8795-4b6078da6508', u'ip_address': u'11.219.1.1'}], id=405063b9-01cc-4a2b-b12f-9839435308bb, mac_address=fa:16:3e:3d:32:92, name=, network_id=c488251b-964b-472c-89c5-f93c510ef717, port_security_enabled=False, project_id=fa8500bb2b5f42e698af7eb9461c6a60, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=fa8500bb2b5f42e698af7eb9461c6a60, updated_at=2018-06-26T09:46:14Z
2018-06-26 11:46:15.528 10812 DEBUG neutron.agent.dhcp.agent [req-71360d0c-3d9d-4a0e-b864-f8a8a4bc1e81 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Calling driver for network: c488251b-964b-472c-89c5-f93c510ef717 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-26 11:46:15.534 10812 DEBUG neutron.agent.linux.dhcp [req-71360d0c-3d9d-4a0e-b864-f8a8a4bc1e81 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Building host file: /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-26 11:46:15.538 10812 DEBUG neutron.agent.linux.dhcp [req-71360d0c-3d9d-4a0e-b864-f8a8a4bc1e81 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Done building host file /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-26 11:46:15.547 10812 DEBUG neutron.agent.linux.utils [req-71360d0c-3d9d-4a0e-b864-f8a8a4bc1e81 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '14888'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:46:15.582 10812 DEBUG neutron.agent.linux.dhcp [req-71360d0c-3d9d-4a0e-b864-f8a8a4bc1e81 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Reloading allocations for network: c488251b-964b-472c-89c5-f93c510ef717 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-26 11:46:15.584 10812 DEBUG neutron.agent.linux.utils [req-71360d0c-3d9d-4a0e-b864-f8a8a4bc1e81 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', '-4', 'route', 'list', 'dev', 'tapf857e23e-8a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:46:15.613 10812 DEBUG neutron.agent.linux.utils [req-71360d0c-3d9d-4a0e-b864-f8a8a4bc1e81 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', '-6', 'route', 'list', 'dev', 'tapf857e23e-8a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:46:15.639 10812 DEBUG neutron.agent.linux.utils [req-71360d0c-3d9d-4a0e-b864-f8a8a4bc1e81 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Unable to access /var/lib/neutron/external/pids/c488251b-964b-472c-89c5-f93c510ef717.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-26 11:46:15.641 10812 DEBUG neutron.agent.linux.utils [req-71360d0c-3d9d-4a0e-b864-f8a8a4bc1e81 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Unable to access /var/lib/neutron/external/pids/c488251b-964b-472c-89c5-f93c510ef717.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-26 11:46:15.642 10812 DEBUG neutron.agent.linux.external_process [req-71360d0c-3d9d-4a0e-b864-f8a8a4bc1e81 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] No process started for c488251b-964b-472c-89c5-f93c510ef717 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-26 11:46:15.642 10812 DEBUG oslo_concurrency.lockutils [req-71360d0c-3d9d-4a0e-b864-f8a8a4bc1e81 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Releasing semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-26 11:46:17.288 10812 DEBUG oslo_concurrency.lockutils [req-71360d0c-3d9d-4a0e-b864-f8a8a4bc1e81 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Acquired semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-26 11:46:17.290 10812 INFO neutron.agent.dhcp.agent [req-71360d0c-3d9d-4a0e-b864-f8a8a4bc1e81 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-26T09:46:13Z, description=, device_id=cba92f07-3ddc-4e6b-98aa-37bb1d876edc, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'dc38aae2-7422-48a5-8795-4b6078da6508', u'ip_address': u'11.219.1.1'}], id=405063b9-01cc-4a2b-b12f-9839435308bb, mac_address=fa:16:3e:3d:32:92, name=, network_id=c488251b-964b-472c-89c5-f93c510ef717, port_security_enabled=False, project_id=fa8500bb2b5f42e698af7eb9461c6a60, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=fa8500bb2b5f42e698af7eb9461c6a60, updated_at=2018-06-26T09:46:14Z
2018-06-26 11:46:17.294 10812 DEBUG neutron.agent.dhcp.agent [req-71360d0c-3d9d-4a0e-b864-f8a8a4bc1e81 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Calling driver for network: c488251b-964b-472c-89c5-f93c510ef717 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-26 11:46:17.299 10812 DEBUG neutron.agent.linux.dhcp [req-71360d0c-3d9d-4a0e-b864-f8a8a4bc1e81 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Building host file: /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-26 11:46:17.305 10812 DEBUG neutron.agent.linux.dhcp [req-71360d0c-3d9d-4a0e-b864-f8a8a4bc1e81 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Done building host file /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-26 11:46:17.315 10812 DEBUG neutron.agent.linux.utils [req-71360d0c-3d9d-4a0e-b864-f8a8a4bc1e81 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '14888'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:46:17.337 10812 DEBUG neutron.agent.linux.dhcp [req-71360d0c-3d9d-4a0e-b864-f8a8a4bc1e81 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Reloading allocations for network: c488251b-964b-472c-89c5-f93c510ef717 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-26 11:46:17.340 10812 DEBUG neutron.agent.linux.utils [req-71360d0c-3d9d-4a0e-b864-f8a8a4bc1e81 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', '-4', 'route', 'list', 'dev', 'tapf857e23e-8a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:46:17.360 10812 DEBUG neutron.agent.linux.utils [req-71360d0c-3d9d-4a0e-b864-f8a8a4bc1e81 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', '-6', 'route', 'list', 'dev', 'tapf857e23e-8a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:46:17.393 10812 DEBUG neutron.agent.linux.utils [req-71360d0c-3d9d-4a0e-b864-f8a8a4bc1e81 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Unable to access /var/lib/neutron/external/pids/c488251b-964b-472c-89c5-f93c510ef717.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-26 11:46:17.394 10812 DEBUG neutron.agent.linux.utils [req-71360d0c-3d9d-4a0e-b864-f8a8a4bc1e81 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Unable to access /var/lib/neutron/external/pids/c488251b-964b-472c-89c5-f93c510ef717.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-26 11:46:17.395 10812 DEBUG neutron.agent.linux.external_process [req-71360d0c-3d9d-4a0e-b864-f8a8a4bc1e81 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] No process started for c488251b-964b-472c-89c5-f93c510ef717 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-26 11:46:17.396 10812 DEBUG oslo_concurrency.lockutils [req-71360d0c-3d9d-4a0e-b864-f8a8a4bc1e81 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Releasing semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-26 11:46:20.488 10812 DEBUG oslo_concurrency.lockutils [req-743a4b14-4b3a-4b8f-be83-8c1bdbe42611 - - - - -] Acquired semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-26 11:46:20.488 10812 INFO neutron.agent.dhcp.agent [req-743a4b14-4b3a-4b8f-be83-8c1bdbe42611 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-26T09:46:13Z, description=, device_id=cba92f07-3ddc-4e6b-98aa-37bb1d876edc, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'dc38aae2-7422-48a5-8795-4b6078da6508', u'ip_address': u'11.219.1.1'}], id=405063b9-01cc-4a2b-b12f-9839435308bb, mac_address=fa:16:3e:3d:32:92, name=, network_id=c488251b-964b-472c-89c5-f93c510ef717, port_security_enabled=False, project_id=fa8500bb2b5f42e698af7eb9461c6a60, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=fa8500bb2b5f42e698af7eb9461c6a60, updated_at=2018-06-26T09:46:16Z
2018-06-26 11:46:20.493 10812 DEBUG neutron.agent.dhcp.agent [req-743a4b14-4b3a-4b8f-be83-8c1bdbe42611 - - - - -] Calling driver for network: c488251b-964b-472c-89c5-f93c510ef717 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-26 11:46:20.495 10812 DEBUG neutron.agent.linux.dhcp [req-743a4b14-4b3a-4b8f-be83-8c1bdbe42611 - - - - -] Building host file: /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-26 11:46:20.500 10812 DEBUG neutron.agent.linux.dhcp [req-743a4b14-4b3a-4b8f-be83-8c1bdbe42611 - - - - -] Done building host file /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-26 11:46:20.505 10812 DEBUG neutron.agent.linux.utils [req-743a4b14-4b3a-4b8f-be83-8c1bdbe42611 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '14888'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:46:20.530 10812 DEBUG neutron.agent.linux.dhcp [req-743a4b14-4b3a-4b8f-be83-8c1bdbe42611 - - - - -] Reloading allocations for network: c488251b-964b-472c-89c5-f93c510ef717 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-26 11:46:20.532 10812 DEBUG neutron.agent.linux.utils [req-743a4b14-4b3a-4b8f-be83-8c1bdbe42611 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', '-4', 'route', 'list', 'dev', 'tapf857e23e-8a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:46:20.550 10812 DEBUG neutron.agent.linux.utils [req-743a4b14-4b3a-4b8f-be83-8c1bdbe42611 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', '-6', 'route', 'list', 'dev', 'tapf857e23e-8a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:46:20.569 10812 DEBUG neutron.agent.linux.utils [req-743a4b14-4b3a-4b8f-be83-8c1bdbe42611 - - - - -] Unable to access /var/lib/neutron/external/pids/c488251b-964b-472c-89c5-f93c510ef717.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-26 11:46:20.570 10812 DEBUG neutron.agent.linux.utils [req-743a4b14-4b3a-4b8f-be83-8c1bdbe42611 - - - - -] Unable to access /var/lib/neutron/external/pids/c488251b-964b-472c-89c5-f93c510ef717.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-26 11:46:20.570 10812 DEBUG neutron.agent.linux.external_process [req-743a4b14-4b3a-4b8f-be83-8c1bdbe42611 - - - - -] No process started for c488251b-964b-472c-89c5-f93c510ef717 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-26 11:46:20.571 10812 DEBUG oslo_concurrency.lockutils [req-743a4b14-4b3a-4b8f-be83-8c1bdbe42611 - - - - -] Releasing semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-26 11:46:21.300 10812 DEBUG oslo_concurrency.lockutils [req-743a4b14-4b3a-4b8f-be83-8c1bdbe42611 - - - - -] Acquired semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-26 11:46:21.307 10812 INFO neutron.agent.dhcp.agent [req-743a4b14-4b3a-4b8f-be83-8c1bdbe42611 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-06-26T09:46:13Z, description=, device_id=cba92f07-3ddc-4e6b-98aa-37bb1d876edc, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'dc38aae2-7422-48a5-8795-4b6078da6508', u'ip_address': u'11.219.1.1'}], id=405063b9-01cc-4a2b-b12f-9839435308bb, mac_address=fa:16:3e:3d:32:92, name=, network_id=c488251b-964b-472c-89c5-f93c510ef717, port_security_enabled=False, project_id=fa8500bb2b5f42e698af7eb9461c6a60, revision_number=6, security_groups=[], status=DOWN, tags=[], tenant_id=fa8500bb2b5f42e698af7eb9461c6a60, updated_at=2018-06-26T09:46:20Z
2018-06-26 11:46:21.308 10812 DEBUG neutron.agent.dhcp.agent [req-743a4b14-4b3a-4b8f-be83-8c1bdbe42611 - - - - -] Calling driver for network: c488251b-964b-472c-89c5-f93c510ef717 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-26 11:46:21.315 10812 DEBUG neutron.agent.linux.dhcp [req-743a4b14-4b3a-4b8f-be83-8c1bdbe42611 - - - - -] Building host file: /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-26 11:46:21.317 10812 DEBUG neutron.agent.linux.dhcp [req-743a4b14-4b3a-4b8f-be83-8c1bdbe42611 - - - - -] Done building host file /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-26 11:46:21.324 10812 DEBUG neutron.agent.linux.utils [req-743a4b14-4b3a-4b8f-be83-8c1bdbe42611 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '14888'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:46:21.343 10812 DEBUG neutron.agent.linux.dhcp [req-743a4b14-4b3a-4b8f-be83-8c1bdbe42611 - - - - -] Reloading allocations for network: c488251b-964b-472c-89c5-f93c510ef717 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-26 11:46:21.345 10812 DEBUG neutron.agent.linux.utils [req-743a4b14-4b3a-4b8f-be83-8c1bdbe42611 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', '-4', 'route', 'list', 'dev', 'tapf857e23e-8a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:46:21.367 10812 DEBUG neutron.agent.linux.utils [req-743a4b14-4b3a-4b8f-be83-8c1bdbe42611 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', '-6', 'route', 'list', 'dev', 'tapf857e23e-8a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:46:21.385 10812 DEBUG neutron.agent.linux.utils [req-743a4b14-4b3a-4b8f-be83-8c1bdbe42611 - - - - -] Unable to access /var/lib/neutron/external/pids/c488251b-964b-472c-89c5-f93c510ef717.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-26 11:46:21.386 10812 DEBUG neutron.agent.linux.utils [req-743a4b14-4b3a-4b8f-be83-8c1bdbe42611 - - - - -] Unable to access /var/lib/neutron/external/pids/c488251b-964b-472c-89c5-f93c510ef717.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-26 11:46:21.388 10812 DEBUG neutron.agent.linux.external_process [req-743a4b14-4b3a-4b8f-be83-8c1bdbe42611 - - - - -] No process started for c488251b-964b-472c-89c5-f93c510ef717 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-26 11:46:21.388 10812 DEBUG oslo_concurrency.lockutils [req-743a4b14-4b3a-4b8f-be83-8c1bdbe42611 - - - - -] Releasing semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-26 11:46:30.364 10812 DEBUG oslo_concurrency.lockutils [req-2374bc35-dec0-491f-aef3-a0ba8c2ceaac 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Acquired semaphore "dhcp-agent-network-lock-c6b5bb99-dcc0-4f71-b6f8-268e791b3287" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-26 11:46:30.365 10812 DEBUG oslo_concurrency.lockutils [req-2374bc35-dec0-491f-aef3-a0ba8c2ceaac 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Releasing semaphore "dhcp-agent-network-lock-c6b5bb99-dcc0-4f71-b6f8-268e791b3287" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-26 11:46:34.487 10812 DEBUG oslo_concurrency.lockutils [req-e4d9afcb-8375-4681-8ec4-e33935df25dd - - - - -] Acquired semaphore "dhcp-agent-network-lock-c6b5bb99-dcc0-4f71-b6f8-268e791b3287" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-26 11:46:34.488 10812 DEBUG oslo_concurrency.lockutils [req-e4d9afcb-8375-4681-8ec4-e33935df25dd - - - - -] Releasing semaphore "dhcp-agent-network-lock-c6b5bb99-dcc0-4f71-b6f8-268e791b3287" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-26 11:46:35.294 10812 DEBUG oslo_concurrency.lockutils [req-e4d9afcb-8375-4681-8ec4-e33935df25dd - - - - -] Acquired semaphore "dhcp-agent-network-lock-c6b5bb99-dcc0-4f71-b6f8-268e791b3287" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-26 11:46:35.294 10812 DEBUG oslo_concurrency.lockutils [req-e4d9afcb-8375-4681-8ec4-e33935df25dd - - - - -] Releasing semaphore "dhcp-agent-network-lock-c6b5bb99-dcc0-4f71-b6f8-268e791b3287" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-26 11:47:00.302 10812 DEBUG oslo_concurrency.lockutils [req-234c2849-2664-48b7-86b0-5e4db0a7a3fb 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Acquired semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-26 11:47:00.303 10812 INFO neutron.agent.dhcp.agent [req-234c2849-2664-48b7-86b0-5e4db0a7a3fb 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-26T09:46:59Z, description=, device_id=28b1e00f-ac86-4f46-991f-d9ab6520a511, device_owner=, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'dc38aae2-7422-48a5-8795-4b6078da6508', u'ip_address': u'11.219.1.12'}], id=09134f68-5e60-43cb-98c6-9ab803009e64, mac_address=fa:16:3e:6e:f8:0f, name=, network_id=c488251b-964b-472c-89c5-f93c510ef717, port_security_enabled=True, project_id=fa8500bb2b5f42e698af7eb9461c6a60, revision_number=3, security_groups=[u'a5c6cdb0-ed9a-4922-b1c1-9ec9d82b6887'], status=DOWN, tags=[], tenant_id=fa8500bb2b5f42e698af7eb9461c6a60, updated_at=2018-06-26T09:46:59Z
2018-06-26 11:47:00.305 10812 DEBUG neutron.agent.dhcp.agent [req-234c2849-2664-48b7-86b0-5e4db0a7a3fb 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Calling driver for network: c488251b-964b-472c-89c5-f93c510ef717 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-26 11:47:00.307 10812 DEBUG neutron.agent.linux.dhcp [req-234c2849-2664-48b7-86b0-5e4db0a7a3fb 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Building host file: /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-26 11:47:00.313 10812 DEBUG neutron.agent.linux.dhcp [req-234c2849-2664-48b7-86b0-5e4db0a7a3fb 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Done building host file /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-26 11:47:00.318 10812 DEBUG neutron.agent.linux.utils [req-234c2849-2664-48b7-86b0-5e4db0a7a3fb 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '14888'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:47:00.336 10812 DEBUG neutron.agent.linux.dhcp [req-234c2849-2664-48b7-86b0-5e4db0a7a3fb 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Reloading allocations for network: c488251b-964b-472c-89c5-f93c510ef717 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-26 11:47:00.337 10812 DEBUG neutron.agent.linux.utils [req-234c2849-2664-48b7-86b0-5e4db0a7a3fb 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', '-4', 'route', 'list', 'dev', 'tapf857e23e-8a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:47:00.351 10812 DEBUG neutron.agent.linux.utils [req-234c2849-2664-48b7-86b0-5e4db0a7a3fb 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', '-6', 'route', 'list', 'dev', 'tapf857e23e-8a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:47:00.369 10812 DEBUG neutron.agent.linux.utils [req-234c2849-2664-48b7-86b0-5e4db0a7a3fb 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Unable to access /var/lib/neutron/external/pids/c488251b-964b-472c-89c5-f93c510ef717.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-26 11:47:00.369 10812 DEBUG neutron.agent.linux.utils [req-234c2849-2664-48b7-86b0-5e4db0a7a3fb 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Unable to access /var/lib/neutron/external/pids/c488251b-964b-472c-89c5-f93c510ef717.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-26 11:47:00.370 10812 DEBUG neutron.agent.linux.external_process [req-234c2849-2664-48b7-86b0-5e4db0a7a3fb 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] No process started for c488251b-964b-472c-89c5-f93c510ef717 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-26 11:47:00.370 10812 DEBUG oslo_concurrency.lockutils [req-234c2849-2664-48b7-86b0-5e4db0a7a3fb 9d2c68a26e624abbafbe6fcc4ba57da2 fa8500bb2b5f42e698af7eb9461c6a60 - - -] Releasing semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-26 11:47:04.397 10812 DEBUG oslo_concurrency.lockutils [req-ccba325e-02ee-46fd-a6e4-4747b0e73101 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Acquired semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-26 11:47:04.398 10812 INFO neutron.agent.dhcp.agent [req-ccba325e-02ee-46fd-a6e4-4747b0e73101 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-26T09:46:59Z, description=, device_id=28b1e00f-ac86-4f46-991f-d9ab6520a511, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'dc38aae2-7422-48a5-8795-4b6078da6508', u'ip_address': u'11.219.1.12'}], id=09134f68-5e60-43cb-98c6-9ab803009e64, mac_address=fa:16:3e:6e:f8:0f, name=, network_id=c488251b-964b-472c-89c5-f93c510ef717, port_security_enabled=True, project_id=fa8500bb2b5f42e698af7eb9461c6a60, revision_number=4, security_groups=[u'a5c6cdb0-ed9a-4922-b1c1-9ec9d82b6887'], status=DOWN, tags=[], tenant_id=fa8500bb2b5f42e698af7eb9461c6a60, updated_at=2018-06-26T09:47:03Z
2018-06-26 11:47:04.400 10812 DEBUG neutron.agent.dhcp.agent [req-ccba325e-02ee-46fd-a6e4-4747b0e73101 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Calling driver for network: c488251b-964b-472c-89c5-f93c510ef717 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-26 11:47:04.402 10812 DEBUG neutron.agent.linux.dhcp [req-ccba325e-02ee-46fd-a6e4-4747b0e73101 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Building host file: /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-26 11:47:04.413 10812 DEBUG neutron.agent.linux.dhcp [req-ccba325e-02ee-46fd-a6e4-4747b0e73101 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Done building host file /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-26 11:47:04.419 10812 DEBUG neutron.agent.linux.utils [req-ccba325e-02ee-46fd-a6e4-4747b0e73101 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '14888'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:47:04.444 10812 DEBUG neutron.agent.linux.dhcp [req-ccba325e-02ee-46fd-a6e4-4747b0e73101 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Reloading allocations for network: c488251b-964b-472c-89c5-f93c510ef717 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-26 11:47:04.445 10812 DEBUG neutron.agent.linux.utils [req-ccba325e-02ee-46fd-a6e4-4747b0e73101 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', '-4', 'route', 'list', 'dev', 'tapf857e23e-8a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:47:04.464 10812 DEBUG neutron.agent.linux.utils [req-ccba325e-02ee-46fd-a6e4-4747b0e73101 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', '-6', 'route', 'list', 'dev', 'tapf857e23e-8a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:47:04.480 10812 DEBUG neutron.agent.linux.utils [req-ccba325e-02ee-46fd-a6e4-4747b0e73101 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Unable to access /var/lib/neutron/external/pids/c488251b-964b-472c-89c5-f93c510ef717.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-26 11:47:04.484 10812 DEBUG neutron.agent.linux.utils [req-ccba325e-02ee-46fd-a6e4-4747b0e73101 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Unable to access /var/lib/neutron/external/pids/c488251b-964b-472c-89c5-f93c510ef717.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-26 11:47:04.485 10812 DEBUG neutron.agent.linux.external_process [req-ccba325e-02ee-46fd-a6e4-4747b0e73101 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] No process started for c488251b-964b-472c-89c5-f93c510ef717 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-26 11:47:04.487 10812 DEBUG oslo_concurrency.lockutils [req-ccba325e-02ee-46fd-a6e4-4747b0e73101 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Releasing semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-26 11:47:05.168 10812 DEBUG oslo_concurrency.lockutils [req-ccba325e-02ee-46fd-a6e4-4747b0e73101 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Acquired semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-26 11:47:05.169 10812 INFO neutron.agent.dhcp.agent [req-ccba325e-02ee-46fd-a6e4-4747b0e73101 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-06-26T09:46:59Z, description=, device_id=28b1e00f-ac86-4f46-991f-d9ab6520a511, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'dc38aae2-7422-48a5-8795-4b6078da6508', u'ip_address': u'11.219.1.12'}], id=09134f68-5e60-43cb-98c6-9ab803009e64, mac_address=fa:16:3e:6e:f8:0f, name=, network_id=c488251b-964b-472c-89c5-f93c510ef717, port_security_enabled=True, project_id=fa8500bb2b5f42e698af7eb9461c6a60, revision_number=5, security_groups=[u'a5c6cdb0-ed9a-4922-b1c1-9ec9d82b6887'], status=DOWN, tags=[], tenant_id=fa8500bb2b5f42e698af7eb9461c6a60, updated_at=2018-06-26T09:47:04Z
2018-06-26 11:47:05.171 10812 DEBUG neutron.agent.dhcp.agent [req-ccba325e-02ee-46fd-a6e4-4747b0e73101 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Calling driver for network: c488251b-964b-472c-89c5-f93c510ef717 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-26 11:47:05.176 10812 DEBUG neutron.agent.linux.dhcp [req-ccba325e-02ee-46fd-a6e4-4747b0e73101 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Building host file: /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-26 11:47:05.179 10812 DEBUG neutron.agent.linux.dhcp [req-ccba325e-02ee-46fd-a6e4-4747b0e73101 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Done building host file /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-26 11:47:05.185 10812 DEBUG neutron.agent.linux.utils [req-ccba325e-02ee-46fd-a6e4-4747b0e73101 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '14888'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:47:05.212 10812 DEBUG neutron.agent.linux.dhcp [req-ccba325e-02ee-46fd-a6e4-4747b0e73101 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Reloading allocations for network: c488251b-964b-472c-89c5-f93c510ef717 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-26 11:47:05.214 10812 DEBUG neutron.agent.linux.utils [req-ccba325e-02ee-46fd-a6e4-4747b0e73101 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', '-4', 'route', 'list', 'dev', 'tapf857e23e-8a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:47:05.233 10812 DEBUG neutron.agent.linux.utils [req-ccba325e-02ee-46fd-a6e4-4747b0e73101 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', '-6', 'route', 'list', 'dev', 'tapf857e23e-8a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:47:05.262 10812 DEBUG neutron.agent.linux.utils [req-ccba325e-02ee-46fd-a6e4-4747b0e73101 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Unable to access /var/lib/neutron/external/pids/c488251b-964b-472c-89c5-f93c510ef717.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-26 11:47:05.263 10812 DEBUG neutron.agent.linux.utils [req-ccba325e-02ee-46fd-a6e4-4747b0e73101 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Unable to access /var/lib/neutron/external/pids/c488251b-964b-472c-89c5-f93c510ef717.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-26 11:47:05.264 10812 DEBUG neutron.agent.linux.external_process [req-ccba325e-02ee-46fd-a6e4-4747b0e73101 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] No process started for c488251b-964b-472c-89c5-f93c510ef717 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-26 11:47:05.265 10812 DEBUG oslo_concurrency.lockutils [req-ccba325e-02ee-46fd-a6e4-4747b0e73101 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Releasing semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-26 11:47:08.646 10812 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-26 11:47:08.648 10812 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-26 11:47:44.473 10812 DEBUG oslo_concurrency.lockutils [req-9401e92e-17c0-4f52-8223-6cb563d586a6 aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Acquired semaphore "dhcp-agent-network-lock-c6b5bb99-dcc0-4f71-b6f8-268e791b3287" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-26 11:47:44.475 10812 DEBUG oslo_concurrency.lockutils [req-9401e92e-17c0-4f52-8223-6cb563d586a6 aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Releasing semaphore "dhcp-agent-network-lock-c6b5bb99-dcc0-4f71-b6f8-268e791b3287" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-26 11:47:45.523 10812 DEBUG oslo_concurrency.lockutils [req-9401e92e-17c0-4f52-8223-6cb563d586a6 aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Acquired semaphore "dhcp-agent-network-lock-c6b5bb99-dcc0-4f71-b6f8-268e791b3287" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-26 11:47:45.524 10812 DEBUG oslo_concurrency.lockutils [req-9401e92e-17c0-4f52-8223-6cb563d586a6 aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Releasing semaphore "dhcp-agent-network-lock-c6b5bb99-dcc0-4f71-b6f8-268e791b3287" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-26 11:48:08.654 10812 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-26 11:48:08.657 10812 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-26 11:49:08.656 10812 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-26 11:49:08.660 10812 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-26 11:50:08.656 10812 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-26 11:50:08.659 10812 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-26 11:51:08.656 10812 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-26 11:51:08.661 10812 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-26 11:52:04.666 10812 DEBUG oslo_concurrency.lockutils [req-c44a9f92-c6c8-4efb-ac1c-cfc199bc57e3 aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Acquired semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-26 11:52:04.669 10812 DEBUG neutron.agent.dhcp.agent [req-c44a9f92-c6c8-4efb-ac1c-cfc199bc57e3 aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Calling driver for network: c488251b-964b-472c-89c5-f93c510ef717 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-26 11:52:04.672 10812 DEBUG neutron.agent.linux.utils [req-c44a9f92-c6c8-4efb-ac1c-cfc199bc57e3 aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'dhcp_release', 'tapf857e23e-8a', '11.219.1.12', 'fa:16:3e:6e:f8:0f'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:52:04.724 10812 DEBUG neutron.agent.linux.dhcp [req-c44a9f92-c6c8-4efb-ac1c-cfc199bc57e3 aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Building host file: /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-26 11:52:04.742 10812 DEBUG neutron.agent.linux.dhcp [req-c44a9f92-c6c8-4efb-ac1c-cfc199bc57e3 aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Done building host file /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-26 11:52:04.753 10812 DEBUG neutron.agent.linux.utils [req-c44a9f92-c6c8-4efb-ac1c-cfc199bc57e3 aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '14888'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:52:04.779 10812 DEBUG neutron.agent.linux.dhcp [req-c44a9f92-c6c8-4efb-ac1c-cfc199bc57e3 aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Reloading allocations for network: c488251b-964b-472c-89c5-f93c510ef717 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-26 11:52:04.780 10812 DEBUG neutron.agent.linux.utils [req-c44a9f92-c6c8-4efb-ac1c-cfc199bc57e3 aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', '-4', 'route', 'list', 'dev', 'tapf857e23e-8a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:52:04.814 10812 DEBUG neutron.agent.linux.utils [req-c44a9f92-c6c8-4efb-ac1c-cfc199bc57e3 aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', '-6', 'route', 'list', 'dev', 'tapf857e23e-8a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:52:04.837 10812 DEBUG neutron.agent.linux.utils [req-c44a9f92-c6c8-4efb-ac1c-cfc199bc57e3 aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Unable to access /var/lib/neutron/external/pids/c488251b-964b-472c-89c5-f93c510ef717.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-26 11:52:04.838 10812 DEBUG neutron.agent.linux.utils [req-c44a9f92-c6c8-4efb-ac1c-cfc199bc57e3 aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Unable to access /var/lib/neutron/external/pids/c488251b-964b-472c-89c5-f93c510ef717.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-26 11:52:04.839 10812 DEBUG neutron.agent.linux.external_process [req-c44a9f92-c6c8-4efb-ac1c-cfc199bc57e3 aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] No process started for c488251b-964b-472c-89c5-f93c510ef717 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-26 11:52:04.839 10812 DEBUG oslo_concurrency.lockutils [req-c44a9f92-c6c8-4efb-ac1c-cfc199bc57e3 aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Releasing semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-26 11:52:08.656 10812 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-26 11:52:08.662 10812 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.005s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-26 11:52:56.466 10812 DEBUG oslo_concurrency.lockutils [req-d706aa93-d54c-4606-990c-0babdbae657d aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Acquired semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-26 11:52:56.467 10812 DEBUG neutron.agent.dhcp.agent [req-d706aa93-d54c-4606-990c-0babdbae657d aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Calling driver for network: c488251b-964b-472c-89c5-f93c510ef717 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-26 11:52:56.470 10812 DEBUG neutron.agent.linux.utils [req-d706aa93-d54c-4606-990c-0babdbae657d aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'dhcp_release', 'tapf857e23e-8a', '11.219.1.1', 'fa:16:3e:3d:32:92'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:52:56.490 10812 DEBUG neutron.agent.linux.dhcp [req-d706aa93-d54c-4606-990c-0babdbae657d aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Building host file: /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-26 11:52:56.495 10812 DEBUG neutron.agent.linux.dhcp [req-d706aa93-d54c-4606-990c-0babdbae657d aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Done building host file /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-26 11:52:56.503 10812 DEBUG neutron.agent.linux.utils [req-d706aa93-d54c-4606-990c-0babdbae657d aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '14888'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:52:56.520 10812 DEBUG neutron.agent.linux.dhcp [req-d706aa93-d54c-4606-990c-0babdbae657d aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Reloading allocations for network: c488251b-964b-472c-89c5-f93c510ef717 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-26 11:52:56.522 10812 DEBUG neutron.agent.linux.utils [req-d706aa93-d54c-4606-990c-0babdbae657d aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', '-4', 'route', 'list', 'dev', 'tapf857e23e-8a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:52:56.539 10812 DEBUG neutron.agent.linux.utils [req-d706aa93-d54c-4606-990c-0babdbae657d aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', '-6', 'route', 'list', 'dev', 'tapf857e23e-8a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:52:56.554 10812 DEBUG neutron.agent.linux.utils [req-d706aa93-d54c-4606-990c-0babdbae657d aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Unable to access /var/lib/neutron/external/pids/c488251b-964b-472c-89c5-f93c510ef717.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-26 11:52:56.555 10812 DEBUG neutron.agent.linux.utils [req-d706aa93-d54c-4606-990c-0babdbae657d aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Unable to access /var/lib/neutron/external/pids/c488251b-964b-472c-89c5-f93c510ef717.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-26 11:52:56.556 10812 DEBUG neutron.agent.linux.external_process [req-d706aa93-d54c-4606-990c-0babdbae657d aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] No process started for c488251b-964b-472c-89c5-f93c510ef717 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-26 11:52:56.557 10812 DEBUG oslo_concurrency.lockutils [req-d706aa93-d54c-4606-990c-0babdbae657d aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Releasing semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-26 11:53:08.656 10812 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-26 11:53:08.659 10812 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-26 11:53:25.203 10812 DEBUG oslo_concurrency.lockutils [req-a71364e1-b211-4578-bd2a-a2b12eadab8d aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Acquired semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-26 11:53:25.205 10812 DEBUG neutron.agent.dhcp.agent [req-a71364e1-b211-4578-bd2a-a2b12eadab8d aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Calling driver for network: c488251b-964b-472c-89c5-f93c510ef717 action: disable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-26 11:53:25.208 10812 DEBUG neutron.agent.linux.utils [req-a71364e1-b211-4578-bd2a-a2b12eadab8d aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Running command (rootwrap daemon): ['kill', '-9', '14888'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:53:25.231 10812 DEBUG neutron.agent.linux.utils [req-a71364e1-b211-4578-bd2a-a2b12eadab8d aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-06-26 11:53:25.278 10812 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=br-int, port=tapf857e23e-8a) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-26 11:53:25.388 10812 DEBUG neutron.agent.linux.utils [req-c02ad38a-fcae-4157-9c6d-042838d5057e - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:53:25.416 10812 DEBUG neutron.agent.linux.utils [req-c02ad38a-fcae-4157-9c6d-042838d5057e - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'delete', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:53:25.434 10812 DEBUG oslo_concurrency.lockutils [req-c02ad38a-fcae-4157-9c6d-042838d5057e - - - - -] Releasing semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-26 11:53:29.027 10812 DEBUG neutron.agent.dhcp.agent [req-37cb7e6d-98a7-4ab2-9a09-2a445a4ffefe - - - - -] resync (c488251b-964b-472c-89c5-f93c510ef717): ['Agent port was deleted'] _periodic_resync_helper /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:255
2018-06-26 11:53:29.029 10812 INFO neutron.agent.dhcp.agent [req-37cb7e6d-98a7-4ab2-9a09-2a445a4ffefe - - - - -] Synchronizing state
2018-06-26 11:53:29.850 10812 INFO neutron.agent.dhcp.agent [req-f583eaaa-ff30-4f72-a884-f22103e7f598 - - - - -] All active networks have been fetched through RPC.
2018-06-26 11:53:29.852 10812 INFO neutron.agent.dhcp.agent [-] Starting network c488251b-964b-472c-89c5-f93c510ef717 dhcp configuration
2018-06-26 11:53:29.852 10812 DEBUG neutron.agent.dhcp.agent [-] Calling driver for network: c488251b-964b-472c-89c5-f93c510ef717 action: enable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-26 11:53:29.853 10812 DEBUG neutron.agent.linux.utils [-] Unable to access /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-26 11:53:29.854 10812 DEBUG neutron.agent.linux.dhcp [-] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-c488251b-964b-472c-89c5-f93c510ef717 on network c488251b-964b-472c-89c5-f93c510ef717 does not yet exist. Checking for a reserved port. _setup_reserved_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1267
2018-06-26 11:53:29.854 10812 DEBUG neutron.agent.linux.dhcp [-] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-c488251b-964b-472c-89c5-f93c510ef717 on network c488251b-964b-472c-89c5-f93c510ef717 does not yet exist. Creating new one. _setup_new_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1288
2018-06-26 11:53:29.861 10812 INFO neutron.agent.dhcp.agent [-] Starting network c6b5bb99-dcc0-4f71-b6f8-268e791b3287 dhcp configuration
2018-06-26 11:53:29.861 10812 INFO neutron.agent.dhcp.agent [-] Finished network c6b5bb99-dcc0-4f71-b6f8-268e791b3287 dhcp configuration
2018-06-26 11:53:32.615 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:53:32.632 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'add', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:53:32.647 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'sysctl', '-w', 'net.ipv4.conf.all.promote_secondaries=1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:53:32.677 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', 'link', 'set', 'lo', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:53:32.688 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'sysctl', '-w', 'net.ipv6.conf.default.accept_ra=0'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:53:32.707 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', 'link', 'set', 'tap5835845f-e2', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:53:32.731 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', '-o', 'link', 'show', 'tap5835845f-e2'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:53:32.748 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-06-26 11:53:32.771 10812 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=None, port=tap5835845f-e2) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-26 11:53:32.773 10812 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Transaction caused no change do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:114
2018-06-26 11:53:32.774 10812 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): AddPortCommand(bridge=br-int, may_exist=False, port=tap5835845f-e2) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-26 11:53:32.776 10812 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=1): DbSetCommand(table=Interface, col_values=(('type', 'internal'), ('external_ids', {'iface-id': u'5835845f-e296-4add-a46e-d2a3252e439f', 'iface-status': 'active', 'attached-mac': u'fa:16:3e:53:d1:cd'})), record=tap5835845f-e2) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-26 11:53:32.813 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tap5835845f-e2', 'address', 'fa:16:3e:53:d1:cd'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:53:32.823 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:53:32.834 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tap5835845f-e2', 'netns', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:53:32.863 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', 'link', 'set', 'tap5835845f-e2', 'mtu', '1450'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:53:32.880 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', 'link', 'set', 'tap5835845f-e2', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:53:32.897 10812 DEBUG oslo_concurrency.lockutils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Acquired semaphore "iptables-qdhcp-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-26 11:53:32.898 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'iptables-save'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:53:32.925 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'iptables-restore', '-n'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:53:32.947 10812 DEBUG neutron.agent.linux.iptables_manager [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] IPTablesManager.apply completed with success. 56 iptables commands were issued _apply_synchronized /usr/lib/python2.7/site-packages/neutron/agent/linux/iptables_manager.py:601
2018-06-26 11:53:32.948 10812 DEBUG oslo_concurrency.lockutils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Releasing semaphore "iptables-qdhcp-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-26 11:53:32.949 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', '-o', 'link', 'show', 'tap5835845f-e2'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:53:32.963 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', 'addr', 'show', 'tap5835845f-e2'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:53:32.979 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', '-4', 'addr', 'add', '11.219.1.2/24', 'scope', 'global', 'dev', 'tap5835845f-e2', 'brd', '11.219.1.255'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:53:32.992 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', '-4', 'route', 'list', 'dev', 'tap5835845f-e2'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:53:33.007 10812 DEBUG neutron.agent.linux.dhcp [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Setting IPv4 gateway for dhcp netns on net c488251b-964b-472c-89c5-f93c510ef717 to 11.219.1.1 _set_default_route_ip_version /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1164
2018-06-26 11:53:33.009 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', '-4', 'route', 'replace', 'default', 'via', '11.219.1.1', 'dev', 'tap5835845f-e2'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:53:33.023 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', '-6', 'route', 'list', 'dev', 'tap5835845f-e2'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:53:33.038 10812 DEBUG neutron.agent.linux.dhcp [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Cleaning stale devices for network c488251b-964b-472c-89c5-f93c510ef717 _cleanup_stale_devices /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1369
2018-06-26 11:53:33.039 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:53:33.052 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'find', '/sys/class/net', '-maxdepth', '1', '-type', 'l', '-printf', '%f '] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:53:33.071 10812 DEBUG neutron.agent.linux.dhcp [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Building initial lease file: /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/leases _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:623
2018-06-26 11:53:33.073 10812 DEBUG neutron.agent.linux.dhcp [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Done building initial lease file /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/leases with contents:
1530093213 fa:16:3e:53:d1:cd 11.219.1.2 * *
 _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:651
2018-06-26 11:53:33.074 10812 DEBUG neutron.agent.linux.dhcp [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Building host file: /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-26 11:53:33.076 10812 DEBUG neutron.agent.linux.dhcp [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Done building host file /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-26 11:53:33.078 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Unable to access /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-26 11:53:33.080 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'dnsmasq', '--no-hosts', '--no-resolv', '--strict-order', '--except-interface=lo', '--pid-file=/var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/pid', '--dhcp-hostsfile=/var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/host', '--addn-hosts=/var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/addn_hosts', '--dhcp-optsfile=/var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/opts', '--dhcp-leasefile=/var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/leases', '--dhcp-match=set:ipxe,175', '--bind-interfaces', '--interface=tap5835845f-e2', '--dhcp-range=set:tag0,11.219.1.0,static,255.255.255.0,86400s', '--dhcp-option-force=option:mtu,1450', '--dhcp-lease-max=256', '--conf-file=', '--domain=openstacklocal'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:53:33.117 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Unable to access /var/lib/neutron/external/pids/c488251b-964b-472c-89c5-f93c510ef717.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-26 11:53:33.118 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Unable to access /var/lib/neutron/external/pids/c488251b-964b-472c-89c5-f93c510ef717.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-26 11:53:33.119 10812 DEBUG neutron.agent.linux.external_process [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] No process started for c488251b-964b-472c-89c5-f93c510ef717 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-26 11:53:33.120 10812 INFO neutron.agent.dhcp.agent [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Finished network c488251b-964b-472c-89c5-f93c510ef717 dhcp configuration
2018-06-26 11:53:33.123 10812 INFO neutron.agent.dhcp.agent [req-f583eaaa-ff30-4f72-a884-f22103e7f598 - - - - -] Synchronizing state complete
2018-06-26 11:53:33.133 10812 DEBUG oslo_concurrency.lockutils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Acquired semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-26 11:53:33.134 10812 INFO neutron.agent.dhcp.agent [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-26T09:53:30Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-c488251b-964b-472c-89c5-f93c510ef717, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'dc38aae2-7422-48a5-8795-4b6078da6508', u'ip_address': u'11.219.1.2'}], id=5835845f-e296-4add-a46e-d2a3252e439f, mac_address=fa:16:3e:53:d1:cd, name=, network_id=c488251b-964b-472c-89c5-f93c510ef717, port_security_enabled=False, project_id=fa8500bb2b5f42e698af7eb9461c6a60, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=fa8500bb2b5f42e698af7eb9461c6a60, updated_at=2018-06-26T09:53:30Z
2018-06-26 11:53:33.136 10812 DEBUG neutron.agent.dhcp.agent [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Calling driver for network: c488251b-964b-472c-89c5-f93c510ef717 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-26 11:53:33.137 10812 DEBUG neutron.agent.linux.dhcp [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Building host file: /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-26 11:53:33.140 10812 DEBUG neutron.agent.linux.dhcp [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Done building host file /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-26 11:53:33.145 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '18300'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:53:33.161 10812 DEBUG neutron.agent.linux.dhcp [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Reloading allocations for network: c488251b-964b-472c-89c5-f93c510ef717 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-26 11:53:33.163 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', '-4', 'route', 'list', 'dev', 'tap5835845f-e2'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:53:33.178 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', '-6', 'route', 'list', 'dev', 'tap5835845f-e2'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:53:33.195 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Unable to access /var/lib/neutron/external/pids/c488251b-964b-472c-89c5-f93c510ef717.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-26 11:53:33.196 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Unable to access /var/lib/neutron/external/pids/c488251b-964b-472c-89c5-f93c510ef717.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-26 11:53:33.197 10812 DEBUG neutron.agent.linux.external_process [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] No process started for c488251b-964b-472c-89c5-f93c510ef717 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-26 11:53:33.198 10812 DEBUG oslo_concurrency.lockutils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Releasing semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-26 11:53:33.199 10812 DEBUG oslo_concurrency.lockutils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Acquired semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-26 11:53:33.200 10812 INFO neutron.agent.dhcp.agent [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-06-26T09:53:30Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-c488251b-964b-472c-89c5-f93c510ef717, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'dc38aae2-7422-48a5-8795-4b6078da6508', u'ip_address': u'11.219.1.2'}], id=5835845f-e296-4add-a46e-d2a3252e439f, mac_address=fa:16:3e:53:d1:cd, name=, network_id=c488251b-964b-472c-89c5-f93c510ef717, port_security_enabled=False, project_id=fa8500bb2b5f42e698af7eb9461c6a60, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=fa8500bb2b5f42e698af7eb9461c6a60, updated_at=2018-06-26T09:53:31Z
2018-06-26 11:53:33.202 10812 DEBUG neutron.agent.dhcp.agent [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Calling driver for network: c488251b-964b-472c-89c5-f93c510ef717 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-26 11:53:33.205 10812 DEBUG neutron.agent.linux.dhcp [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Building host file: /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-26 11:53:33.208 10812 DEBUG neutron.agent.linux.dhcp [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Done building host file /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-26 11:53:33.213 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '18300'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:53:33.239 10812 DEBUG neutron.agent.linux.dhcp [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Reloading allocations for network: c488251b-964b-472c-89c5-f93c510ef717 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-26 11:53:33.242 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', '-4', 'route', 'list', 'dev', 'tap5835845f-e2'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:53:33.264 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717', 'ip', '-6', 'route', 'list', 'dev', 'tap5835845f-e2'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:53:33.283 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Unable to access /var/lib/neutron/external/pids/c488251b-964b-472c-89c5-f93c510ef717.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-26 11:53:33.284 10812 DEBUG neutron.agent.linux.utils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Unable to access /var/lib/neutron/external/pids/c488251b-964b-472c-89c5-f93c510ef717.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-26 11:53:33.286 10812 DEBUG neutron.agent.linux.external_process [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] No process started for c488251b-964b-472c-89c5-f93c510ef717 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-26 11:53:33.287 10812 DEBUG oslo_concurrency.lockutils [req-7466c1fe-a62d-4eae-93f2-c9942d98a145 - - - - -] Releasing semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-26 11:53:36.004 10812 DEBUG oslo_concurrency.lockutils [req-398ca368-4475-480f-b1c8-1e23223526ec aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Acquired semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-26 11:53:36.005 10812 DEBUG neutron.agent.dhcp.agent [req-398ca368-4475-480f-b1c8-1e23223526ec aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Calling driver for network: c488251b-964b-472c-89c5-f93c510ef717 action: disable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-26 11:53:36.008 10812 DEBUG neutron.agent.linux.utils [req-398ca368-4475-480f-b1c8-1e23223526ec aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Running command (rootwrap daemon): ['kill', '-9', '18300'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:53:36.026 10812 DEBUG neutron.agent.linux.utils [req-398ca368-4475-480f-b1c8-1e23223526ec aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-06-26 11:53:36.055 10812 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=br-int, port=tap5835845f-e2) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-26 11:53:36.133 10812 DEBUG neutron.agent.linux.utils [req-24aca715-0057-41fd-902b-616576e4fd19 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:53:36.145 10812 DEBUG neutron.agent.linux.utils [req-24aca715-0057-41fd-902b-616576e4fd19 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'delete', 'qdhcp-c488251b-964b-472c-89c5-f93c510ef717'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:53:36.159 10812 DEBUG oslo_concurrency.lockutils [req-24aca715-0057-41fd-902b-616576e4fd19 - - - - -] Releasing semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-26 11:53:37.613 10812 DEBUG oslo_concurrency.lockutils [req-398ca368-4475-480f-b1c8-1e23223526ec aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Acquired semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-26 11:53:38.082 10812 DEBUG neutron.agent.linux.utils [req-b359a24c-07d1-4d69-b1de-3785c9348705 - - - - -] Unable to access /var/lib/neutron/external/pids/c488251b-964b-472c-89c5-f93c510ef717.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-26 11:53:38.084 10812 DEBUG neutron.agent.linux.utils [req-b359a24c-07d1-4d69-b1de-3785c9348705 - - - - -] Unable to access /var/lib/neutron/external/pids/c488251b-964b-472c-89c5-f93c510ef717.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-26 11:53:38.085 10812 DEBUG neutron.agent.linux.external_process [req-b359a24c-07d1-4d69-b1de-3785c9348705 - - - - -] No process started for c488251b-964b-472c-89c5-f93c510ef717 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-26 11:53:38.086 10812 DEBUG neutron.agent.dhcp.agent [req-b359a24c-07d1-4d69-b1de-3785c9348705 - - - - -] Calling driver for network: c488251b-964b-472c-89c5-f93c510ef717 action: disable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-26 11:53:38.088 10812 DEBUG neutron.agent.linux.utils [req-b359a24c-07d1-4d69-b1de-3785c9348705 - - - - -] Unable to access /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-26 11:53:38.089 10812 DEBUG neutron.agent.linux.utils [req-b359a24c-07d1-4d69-b1de-3785c9348705 - - - - -] Unable to access /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-26 11:53:38.090 10812 DEBUG neutron.agent.linux.external_process [req-b359a24c-07d1-4d69-b1de-3785c9348705 - - - - -] No process started for c488251b-964b-472c-89c5-f93c510ef717 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-26 11:53:38.091 10812 DEBUG neutron.agent.linux.dhcp [req-b359a24c-07d1-4d69-b1de-3785c9348705 - - - - -] Impossibile accedere a /var/lib/neutron/dhcp/c488251b-964b-472c-89c5-f93c510ef717/interface _get_value_from_conf_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:268
2018-06-26 11:53:38.092 10812 DEBUG neutron.agent.linux.dhcp [req-b359a24c-07d1-4d69-b1de-3785c9348705 - - - - -] No interface exists for network c488251b-964b-472c-89c5-f93c510ef717 destroy /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1482
2018-06-26 11:53:38.149 10812 DEBUG neutron.agent.linux.utils [req-43a8ca53-dbbc-4785-9cc5-9787fef70f4b - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-26 11:53:38.167 10812 DEBUG neutron.agent.linux.dhcp [req-43a8ca53-dbbc-4785-9cc5-9787fef70f4b - - - - -] Namespace already deleted: qdhcp-c488251b-964b-472c-89c5-f93c510ef717 _destroy_namespace_and_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:246
2018-06-26 11:53:38.168 10812 DEBUG oslo_concurrency.lockutils [req-43a8ca53-dbbc-4785-9cc5-9787fef70f4b - - - - -] Releasing semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-26 11:53:39.423 10812 DEBUG oslo_concurrency.lockutils [req-398ca368-4475-480f-b1c8-1e23223526ec aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Acquired semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-26 11:53:39.424 10812 DEBUG oslo_concurrency.lockutils [req-398ca368-4475-480f-b1c8-1e23223526ec aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Releasing semaphore "dhcp-agent-network-lock-c488251b-964b-472c-89c5-f93c510ef717" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-26 11:54:03.132 10812 DEBUG neutron.agent.dhcp.agent [req-f583eaaa-ff30-4f72-a884-f22103e7f598 - - - - -] resync (c488251b-964b-472c-89c5-f93c510ef717): ['Agent port was deleted'] _periodic_resync_helper /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:255
2018-06-26 11:54:03.135 10812 INFO neutron.agent.dhcp.agent [req-f583eaaa-ff30-4f72-a884-f22103e7f598 - - - - -] Synchronizing state
2018-06-26 11:54:04.017 10812 INFO neutron.agent.dhcp.agent [req-8a4563bc-2bfd-4a35-99d6-3b7787f2be8a - - - - -] All active networks have been fetched through RPC.
2018-06-26 11:54:04.019 10812 INFO neutron.agent.dhcp.agent [-] Starting network c6b5bb99-dcc0-4f71-b6f8-268e791b3287 dhcp configuration
2018-06-26 11:54:04.019 10812 INFO neutron.agent.dhcp.agent [-] Finished network c6b5bb99-dcc0-4f71-b6f8-268e791b3287 dhcp configuration
2018-06-26 11:54:04.020 10812 INFO neutron.agent.dhcp.agent [req-8a4563bc-2bfd-4a35-99d6-3b7787f2be8a - - - - -] Synchronizing state complete
2018-06-26 11:54:08.665 10812 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-26 11:54:08.668 10812 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
